### PR TITLE
MOTECH-2275: Remove forms with wrong connecting settings 

### PR DESCRIPTION
**Pull request by DamianNowakSoldevelo**

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by sebbrudzinski** [comment]
test this please

**Comment by ngraczewski** [comment]
Looks good to me.
@sebbrudzinski Can you take a look at this?

File: commcare/src/main/java/org/motechproject/commcare/config/manager/ConfigurationManager.java [hunk 1]
```diff
@@ -39,8 +39,17 @@ public void configCreated(String configName) {
 
     @Transactional
     public void configUpdated(String configName) {
+        configUpdated(configName, true);
+    }
+
+    @Transactional
+    public void configUpdated(String configName, boolean downloadApplications) {
         LOGGER.info("Configuration [{}] updated, fetching Commcare schema, {}", configName);
```
Comment (sebbrudzinski) [inline commcare/src/main/java/org/motechproject/commcare/config/manager/ConfigurationManager.java]: Move the log inside the if and log something in else

**Comment by sebbrudzinski** [comment]
test this please

**Comment by sebbrudzinski** [comment]
@pgesek LGTM

**Pull request merged**